code/modules/antagonists/eldritch_cult/eldritch_items.dm

```diff
@@ -0,0 +1,124 @@
+	. = ..()
+	if(!IS_HERETIC(user))
+		return
+	if(!target)
+		to_chat(user,"<span class='warning'>No target could be found. Put the living heart on the rune and use the rune to recieve a target.</span>")
+		return
+	var/dist = get_dist(user.loc,target.loc)
+	var/dir = get_dir(user.loc,target.loc)
+
+	switch(dist)
+		if(0 to 15)
+			to_chat(user,"<span class='warning'>[target.real_name] is near you. They are to the [dir2text(dir)] of you!</span>")
+		if(16 to 31)
+			to_chat(user,"<span class='warning'>[target.real_name] is somewhere in your vicinty. They are to the [dir2text(dir)] of you!</span>")
+		if(32 to 127)
+			to_chat(user,"<span class='warning'>[target.real_name] is far away from you. They are to the [dir2text(dir)] of you!</span>")
+		else
+			to_chat(user,"<span class='warning'>[target.real_name] is beyond our reach.</span>")
+
+	if(target.stat == DEAD)
+		to_chat(user,"<span class='warning'>[target.real_name] is dead. Bring them onto a transmutation rune!</span>")
+
+/obj/item/melee/sickly_blade
+	name = "Sickly blade"
+	desc = "Twisted sickle with an ornamental eye. The eyes looks at you."
+	icon = 'icons/obj/eldritch.dmi'
+	icon_state = "eldritch_blade"
+	inhand_icon_state = "eldritch_blade"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	flags_1 = CONDUCT_1
+	sharpness = IS_SHARP
+	w_class = WEIGHT_CLASS_NORMAL
+	force = 17
+	throwforce = 10
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "rended")
+
+/obj/item/melee/sickly_blade/attack(mob/living/M, mob/living/user)
+	if(!IS_HERETIC(user))
+		to_chat(user,"<span class='danger'>Echoes of the breat beyond break your mind!</span>")
+		var/mob/living/carbon/human/human_user = user
+		human_user.AdjustStun(5 SECONDS)
+		return FALSE
+	return ..()
+
+/obj/item/melee/sickly_blade/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	var/datum/antagonist/heretic/cultie = user.mind.has_antag_datum(/datum/antagonist/heretic)
+	if(!cultie)
+		return
+	for(var/X in cultie.get_all_knowledge())
+		var/datum/eldritch_knowledge/eldritch_knowledge_datum = X
+		eldritch_knowledge_datum.on_eldritch_blade(target,user,proximity_flag,click_parameters)
+
+/obj/item/melee/sickly_blade/rust
+	name = "Rusted Blade"
+
+/obj/item/melee/sickly_blade/ash
+	name = "Ashen Blade"
+
+/obj/item/melee/sickly_blade/flesh
+	name = "Flesh Blade"
+
+/obj/item/clothing/neck/eldritch_amulet
+	name = "Warm Eldritch Medallion"
+	desc = "Eldritch medallion that let's you see."
+	icon = 'icons/obj/eldritch.dmi'
+	icon_state = "eye_medalion"
+	w_class = WEIGHT_CLASS_SMALL
+	///What trait do we want to add upon equipiing
+	var/trait = TRAIT_THERMAL_VISION
+
+/obj/item/clothing/neck/eldritch_amulet/equipped(mob/user, slot)
+	. = ..()
+	if(ishuman(user) && user.mind && slot == ITEM_SLOT_NECK && IS_HERETIC(user) )
```

Comment:
aw, I kind of liked being able to sell these to people

well, if you're going to make it heretic-only, you should make it make you see TOO far beyond if you wear it as a non-heretic